release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -92,6 +92,12 @@ The xref:../support/remote_health_monitoring/using-insights-to-identify-issues-w
 [id="ocp-4-7-registry"]
 === Registry
 
+[id="ocp-4-7-registry-oci-support"]
+==== Open Container Initiative images support
+
+The {product-title} Image Registry Operator and Image Pruner now support Open Container Initiative (OCI) images. You can use OCI images in the same way you would use Docker `schema2` images. 
```

Comment:
It would be better if we replace "Image Registry Operator" and "Image Pruner" by "Image Streams" and "Internal Registry".